core/src/09-special-equipment/01-chapter-nine.md

```diff
@@ -840,3 +842,81 @@ ___
 | Giant Scorpion | 4 | 35' | Independent 1 | Agility 5 <br>Perception 5 | Bane Focus I (Persistent Damage) | 20 | 2 |  Guard: 15 <br>Toughness: 15 <br>Resolve: 18 |
 
 </div>
+
+
+
+
+
+
+
+
+
+
+Cody JA Brown
+Dan Scantland
+Daniel Marjoram
+Deven Morgan
+Donna Waldrop
+Elvenoob
+Giuliano Michelon
+Gregg Miller
+John Spearing
+Justin Packwood
+Kerry Wilcox
+KICK in a BOX Inc.
+Latt
+Lone Shark Games
+Marcelino
+Mario
```

Comment:
Is this a different Mario? If not, could I have my last name. :)

Mario Lurig

_Sir Lurig is fine too ;-) j/k_